test/Dockerfile.bionic

```
@@ -57,7 +57,7 @@ RUN install-tool gradle 7.5.1
 RUN install-tool nix 2.3.10
 
 # renovate: datasource=node
-RUN install-tool node v16.18.0
+RUN install-tool node v18.12.0
```

```suggestion
RUN install-tool node v16.18.0
```

v18 isn't supported on bionic
- #362